backend/src/index.js

```diff
@@ -1,3 +1,11 @@
+/**
```

Comment:
Whole file need to be refactored. Extract logical components and add them into different modules in the `startup` directory.